pyproject.toml

```diff
@@ -141,6 +141,7 @@ black = "19.3b0"
 # Latest version compatible with `trio_typing~=0.5`, required by qtrio
 mypy = "v0.910"
 setuptools = "^63.1.0"
+click = "8.0.4"
```

Comment:
We didn't use click in the helper scripts so they can run standalone which make them easier to integrate (and update) in client-specific release repo

Reply:
Also this change is incompatible with what is done in the `.azure-pipelines.yml`

Reply:
I've added it because otherwise `black` don't work on my IDE with the virtualenv

can you give references for: 

> this change is incompatible with what is done in the .azure-pipelines.yml

Because the CI for `Azure-Pipeline` seems fine

Reply:
My bad ! I thought you added click to use it in `freeze_program.py` (hence this script would be required to be launched from a virtual env, which is not done on azure)